.github/workflows/release.yml

```diff
@@ -8,16 +8,19 @@ env:
   DOTNET_NOLOGO: true
 jobs:
   release:
-    runs-on: ubuntu-20.04
+    name: Windows
+    runs-on: windows-2019
     steps:
       - name: Checkout
         uses: actions/checkout@v3.3.0
         with:
-          fetch-depth: 0     
+          fetch-depth: 0
       - name: Setup .NET SDK
         uses: actions/setup-dotnet@v3.0.3
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: |
+            6.0.x
+            3.1.x
```

Comment:
You don't need older runtimes in release workflows. That's only so tests will run on the older framework in CI. Actually the CI workflow should have 3.1 removed as it's out of support.